src/virtual-dom-utils.ls

```diff
@@ -24,7 +24,7 @@ form-elements = (tree, path, input-names) ->
 route-metadata = (tree) ->
   route = extract-route tree
 
-  title = if route.get-title then that! else ""
+  title = if route.get-title then that.call route else ""
```

Comment:
If I recall correctly, livescript lets you bind with `~`.  So, the following should work for you.

```
title = if route~get-title then that! else ""
```